intro-HPC/ch_FAQ.tex

```diff
@@ -0,0 +1,35 @@
+\chapter{Frequently Asked Questions}
+\label{ch:faq}
+
+\section{When will my job start?}
+
+The clusters use a fair-share sheduling policy. There is no guarantee on when a
+job will start, since it depends on a number of factors. One of these factors is
+the priority of the job, which is determined by
+\begin{itemize}
+    \item historical use: the aim is to balance usage over users, so
+        infrequent users get a highter priority
+
+    \item requested resources (amount of cores, walltime, memory, ...): the larger
+        the resource request is, the lower the priority will be
+
+    \item time waiting in queue: queued jobs get a higher priority over time
+
+    \item user limits: this avoids having a single user hog the entire cluster
+
+\end{itemize}
+
+\section{Can I share my account with someone else?}
+
+\strong{NO!}
+\ifgent
+According to \href{https://helpdesk.ugent.be/account/en/regels.php}{the rules},
+this is not allowed.
+\fi % TODO: url of other institutions
+If you want to share data, there are alternatives (like a shared
+directories in VO space).
```

Comment:
refer to sections with more info?

Reply:
I don't really know how VO's work, where to find out about them and where to put the documentation about them. Can we discuss this tomorrow?

Reply:
See http://hpc.ugent.be/userwiki/index.php/User:VSCVos + #194 (but yes, we can discuss this tomorrow, of course).

Reply:
There is a different PR to document VO's: #237